vw_golf_mk4.dbc

```diff
@@ -1138,6 +1138,39 @@ BO_ 210 HCA_1: 5 XXX
  SG_ LM_OffSign : 31|1@1+ (1,0) [0|1] "" XXX
  SG_ Vib_Freq : 32|4@1+ (4,0) [0|60] "Hz" XXX
  SG_ Vib_Amp : 36|4@1+ (0.5,0) [0|7.5] "Nm" XXX
+ 
+BO_ 644 mMotor_Bremse: 6 Motor_Diesel
+ SG_ MOB_Standby : 12|1@1+ (1,0) [0|1] "" Bremse_MK25AESP
```

Comment:
[ example only ]

```suggestion
 SG_ MOB_Standby : 12|1@1+ (1,0) [0|1] ""  XXX
```

Let's lose the network-node source/consumer labels unless and until we're willing to spend the time overhauling the file to make good use of them. No real sense in adding a few PQ25-specific dangling loose references without the other DBC infrastructure for DBC editors to even make use of them. I admit to having done this inconsistently in the past but we'll clean it all up eventually.